File: OOP/1. Class & Objects.py
class Students:        # Class Definition
    Name=None          # Members Declaration
    Math_mark=None
    Physics_mark=None
    Chemistry_mark=None

    def cutoff(self):         # Function Declaration
        cutoff_mark=(self.Math_mark+self.Physics_mark+self.Chemistry_mark)/3
        return cutoff_mark


Student_1=Students()        #  Object Creation
Student_1.Name="Alex"
Student_1.Math_mark=75
Student_1.Physics_mark=83
Student_1.Chemistry_mark=93

Student_2=Students()
Student_2.Name="Brock"
Student_2.Math_mark=77
Student_2.Physics_mark=62
Student_2.Chemistry_mark=58

print(Student_2.cutoff()) # Function calling using object

File: OOP/1. Class & Objects.cpp
#include<iostream>
using namespace std;

class Students{
    string name;
    float math_mark,phys_mark,chem_mark,cutoff;

  public:
    Students(string name, float math_mark, float phys_mark, float chem_mark){
        this->name=name;
        this->math_mark=math_mark;
        this->phys_mark=phys_mark;
        this->chem_mark=chem_mark;

    }


    void cut(){
        cutoff=math_mark+phys_mark+chem_mark;
        cutoff=cutoff/3;
        cout<<cutoff  ;
    }

};


int main(){
    Students s1("chris",95.6,98,97);
    s1.cut();
    return 0;
}